sdk/schemaregistry/schema-registry-avro/src/models.ts

```diff
@@ -61,33 +61,3 @@ export interface DeserializeOptions {
    */
   schema?: string;
 }
-
-/**
- * A custom error type for failed Avro serialization/deserialization.
- */
-export class AvroError extends Error {
-  /**
-   * The inner error that was thrown by the Avro implementation library.
-   */
-  public innerError?: unknown;
-  /**
-   * The schema ID of the schema that was used during the failing operation.
-   */
-  public schemaId?: string;
```

Comment:
does it matter to recovery scenarios that we no longer are including a schemaId property?

Reply:
It could be the feature crew reversed course and want to wait before supporting this scenario based on what we hear from customers. /cc @JoshLove-msft

Reply:
The hypothesis is that there aren't any recoverable scenarios where the user can do something with the schema ID programmatically. We should still include the schema ID in the exception message for troubleshooting purposes.